### ISSUE TITLE
Latest release broken

### ISSUE THREAD

**Issue by pdxwolfy** [lexicalunit/multi-wrap-guide#24]
After installing the 0.26.0 update in Atom 1.12.1, I no longer have a wrap guide, and can't seem to display any. The context menu for Multi Wrap Guide appear and are active, but nothing happens when I try to create a horizontal or vertical guide.

I also disabled my styles.less file in case that was interfering, but to no avail.

**Comment by pdxwolfy** [comment]
Reinstalled 0.25.2 to get my wrap guide back.

**Comment by lexicalunit** [comment]
Ugh. Atom is changing how they handle shadow elements in the upcoming releases. I didn't realize it was too early to switch over to the new way of doing things. I'll make a patch version to fix it and only remove the deprecated code once Atom v1.13.0 is released. I didn't notice things were broken myself because I'm running the Atom beta on my machine. My apologies!

**Comment by pdxwolfy** [comment]
Okaye-doke. np. Thanx.

**Comment by lexicalunit** [comment]
Alright, just published v0.26.1. I'll close this issue now but please reopen if this doesn't resolve the issue! Thanks for bringing this to my attention 👍

**Comment by pdxwolfy** [comment]
Unfortunately, 0.26.1 does the same thing. I'm back again to 0.25.2, @lexicalunit

**Comment by lexicalunit** [comment]
Wow... I can't believe I just committed that code. Yeah, I see exactly the problem. Sigh. Sorry, I am not operating at 100% today, apparently. I'm gonna create a PR and verify the fix in Travis CI before merging and closing this ticket... which is what I should have done in the first place. Apologies again, I'll get this fixed asap.

**Comment by lexicalunit** [comment]
That was more complicated than I thought. I followed the suggestion by [this Atom blog](http://blog.atom.io/2016/11/14/removing-shadow-dom-boundary-from-text-editor-elements.html) that suggested upping the `engines` version to prevent exactly this kind of issue. I think I have everything working correctly now. I also unpublished the broken versions.

Continue using v0.25.2 for now and hopefully whenever Atom 1.13 is released you'll be able to update to v0.28.0.
